src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValuesMas = Object.values(robot);
+  const originalRobotValues = [];
+
+  for (const curr of robotValuesMas) {
+    if (originalRobotValues.includes(curr)) {
```

Comment:
let's make it simple
you don't actually have to create a separate array to check what's inside your object
for example, you can use `hasOwnProperty` method